site/docs/samples/dashboard-sample.md

```diff
@@ -76,47 +72,46 @@ helm install rel1-backend-server ./tmp/backend_server
+The identifier (name) of the asset reflects the URL of the end point, and is of the form: `<namespace>/<name>`, where backslash characters ("/") in the URL are replaced by a period (".") in the asset name.
+For example, the name of the asset in file asset_start_robot.yaml, represented by the URL, <HOST>/api/control/start-robot, is 
 `fogprotect/api.control.start-robot`.  
 
-## Create JWT authentication key
+## Create the JWT authentication key
 
-As the HTTP requests should contain the role of the user in the header, the dashboard application uses JWT to pass 
-the JWT of the relevant role in the header. The JWT is authenticated using a secret key that we store as a secret 
+As the HTTP requests from the dashboard should contain the role of the logged in user in the header, the dashboard application uses JWT to encode
+the relevant role in the header. The JWT is authenticated using a secret key that we store as a secret 
 in the cluster, in both `fogprotect` and `fybrik-blueprints` namespaces.  
 Create the JWT secret:  
 ```bash
 kubectl apply -f https://raw.githubusercontent.com/fybrik/fogProtect-dashboard-sample/main/secrets/jwt_key_secret.yaml
 kubectl apply -n fybrik-blueprints -f https://raw.githubusercontent.com/fybrik/fogProtect-dashboard-sample/main/secrets/jwt_key_secret.yaml
 ```
 
-## Fybrik manager RBAC
+## Fybrik Manager RBAC
 
-In order for the fybrik manager to be able to access the assets and to deploy the fybrik module that we created, give 
-the manager the relevant [RBAC authorization](https://github.com/fybrik/fogProtect-dashboard-sample/blob/main/fybrik-system-manager-rbac.yaml):  
+In order for the Fybrik Control Plane to be able to access the JWT key that we created, give 
+the Manager the relevant [RBAC authorization](https://github.com/fybrik/fogProtect-dashboard-sample/blob/main/fybrik-system-manager-rbac.yaml):  
 ```bash
 kubectl apply -n fybrik-system -f https://raw.githubusercontent.com/fybrik/fogProtect-dashboard-sample/main/fybrik-system-manager-rbac.yaml
 ```  
-
 ## Deploy the module
 
 In our case, the module as we described earlier is responsible for intercepting the HTTP requests received 
-from the user. Once a request is received a decision must be made regarding the request, it should be either allowed, 
-columns redacted or blocked depending on the role of the user. The decision is made using 
-[OpenPolicyAgent](https://www.openpolicyagent.org), and applying the policy described in 
-[About the dashboard application](#About the dashboard application) and specified 
+from the user. Once a request is received a decision must be made regarding the request based on the role of the user and the characterization of the data being requested; it should be either allowed, potentially with fields redacted, or blocked. The decision is made using 
+[OpenPolicyAgent](https://www.openpolicyagent.org), and applying the policy specified 
```

Comment:
Please add in-line an example policy that will relate to the tag in the in-line asset you have added earlier.

Reply:
True, but having the policy inline will help understand how the policy affects the data.
A link by itself does not explain the policy.
The text here should be helpful in explaining how to write policies as well.